docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
+MAINTAINER svanoort <samvanoort@gmail.com>
+
+
+# Dependencies plus vim for convenience, wget if you're trying to get fancy
+RUN apt-get update \
+  && apt-get install -y make unzip devscripts debhelper rpm expect createrepo ruby maven openjdk-7-jre \
+  && apt-get install -y vim git \
+  && apt-get install -y wget \ 
```

Comment:
@oleg-nenashev If we hit issues with these we could add 'em, but there haven't been any problems so far (the packaging image isn't used for full jenkins run/test, just assembling native packages).   Er, I'm also trying not to lump in too many more packages since the image is already 830 MB.